### ISSUE TITLE
reflection warning: call to method set can't be resolved (target class is unknown)

### ISSUE THREAD

**Issue by daaku** [aroemers/mount-lite#19]
Inside `with-session`: https://github.com/aroemers/mount-lite/blob/5961ac83093eecea0ccbb8c756c1c35792b8e33b/src/mount/lite.clj#L205

I'm getting a this reflection warning:

```
call to method set can't be resolved (target class is unknown)
```

**Comment by aroemers** [comment]
Hi @daaku,

Thank you for your issue report. It should be fixed in 2.1.4.

**Comment by daaku** [comment]
Thanks for the quick fix. Confirmed it works.
